packages/cxl-ui/src/components/cxl-playbook-accordion.js

```diff
@@ -0,0 +1,163 @@
+import '@conversionxl/cxl-lumo-styles';
+import '@vaadin/vaadin-accordion';
+import '@vaadin/vaadin-checkbox';
+import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/register-styles.js';
+import { registerGlobalStyles } from '@conversionxl/cxl-lumo-styles/src/utils';
+import cxlAdeftAccordionGlobalStyles from '../styles/global/cxl-playbook-accordion-css.js';
+import { CXLVaadinAccordion } from './cxl-vaadin-accordion';
+
+/**
+ * Allows opening multiple panels simultaneously.
+ * Saves panel state, restores on page load.
+ */
+@customElement('cxl-playbook-accordion')
+export class CxlPlaybookAccordion extends CXLVaadinAccordion {
+  constructor() {
+    super();
+    registerStyles('vaadin-accordion-panel', [CxlPlaybookAccordion._getAccordionPanelStyles()]);
+    registerStyles('vaadin-checkbox', [CxlPlaybookAccordion._getVaadinCheckboxStyles()]);
+  }
+
+  static _getAccordionPanelStyles() {
+    return css`
+      [part='summary'] {
+        box-shadow: 0 4px 12px rgba(36, 38, 40, 0.12);
+        padding: 0;
+      }
+      :host(.emptyContent) [part='toggle'] {
+        display: none;
+      }
+      [part='toggle'] {
+        padding: 12px;
+      }
+    `;
+  }
+
+  static _getVaadinCheckboxStyles() {
+    return css`
+      :host([theme~='custom']) [part='checkbox'] {
+        border: 2px solid !important;
+        width: 18px;
+        height: 18px;
+      }
+      :host([theme~='custom']) [part='label'] {
+        display: none;
+      }
+    `;
+  }
+
+  /**
+   * Global styles.
+   */
+  ready() {
+    super.ready();
+
+    // Define and register a style sheet for the <vaadin-text-field> component
+    registerGlobalStyles(cxlAdeftAccordionGlobalStyles, {
+      moduleId: 'cxl-playbook-accordion-global',
+    });
+
+    this._updateCheckboxesStates();
+  }
+
+  /**
+   *
+   * @param {HTMLCollection} items
+   * @private
+   */
+  _saveAccordionState(items) {
+    const storageId = this.getAttribute('id');
+    // Avoid null key.
+    if (storageId) {
+      const stateItems = [];
+
+      items.forEach((value, key) => {
+        stateItems[key] = items[key].opened;
+      });
+
+      localStorage.setItem(storageId, JSON.stringify(stateItems));
+
+      this._updateCSSAndPanelStateToCheckboxesStates();
+
+      this._dispatchCustomEvent(stateItems);
+
+      this._saveCheckboxesState();
+    }
+  }
+
+  _saveCheckboxesState() {
+    const stateCheckboxes = [];
+    const checkboxes = this.querySelectorAll('vaadin-checkbox');
+
+    checkboxes.forEach((value, key) => {
+      const checkbox = checkboxes[key];
+
+      stateCheckboxes[key] =
+        checkbox.hasAttribute('aria-checked') && checkbox.getAttribute('aria-checked') === 'true';
+    });
+
+    const checkboxesStorageId = `${this.getAttribute('id')}_checkboxes`;
+    localStorage.setItem(checkboxesStorageId, JSON.stringify(stateCheckboxes));
+  }
+
+  // Restore accordion panel state.
+  _updateItems(items, opened) {
+    super._updateItems(items, opened);
+  }
```

Comment:
Isn't super's method called by default? Why do we explicitly do this here?

Reply:
Right, I changed it in the previous version but ultimately dropped it.
Removed in: https://github.com/conversionxl/aybolit/pull/87/commits/924605731710fde8a0be77646c55e5ba02cf2938